README.md

```diff
@@ -112,7 +112,7 @@ version of TensorFlow I/O according to the table below:
 
 ### Lint
 
-TensorFlow I/O's code conforms through Pylint, Bazel Buildifier, and Clang Format. The following will check the source code and report any lint issues:
+TensorFlow I/O's code conforms through Pylint, Bazel Buildifier, Clang Format, Black, and Pyupgrade. The following will check the source code and report any lint issues:
```

Comment:
Are these set of linters compatible though?

Reply:
@terrytangyuan I would assume it will be ok. Our goal for pyupgrade is to get the majority of  the code into python3 and remove python2. Once most of the files are cleaned we really don't need 100% for pyupgrade. We could remove the pyupgrade then.